From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Charles Kerr <charles@charleskerr.com>
Date: Mon, 14 Apr 2025 12:49:47 -0500
Subject: refactor: grandfather in AutofillPopupView as a subclass of
 WidgetDelegateView

Add a PassKey for std::make_unique<WidgetDelegateView>() | Add a PassKey for std::make_unique<WidgetDelegateView>()

diff --git a/ui/views/widget/widget_delegate.h b/ui/views/widget/widget_delegate.h
index 3284a8d60b1b6e0ca82fff6e8c2d175ed8c1fcee..d07db44429855fdb99a5a0315ba0ba80d5a7dc13 100644
--- a/ui/views/widget/widget_delegate.h
+++ b/ui/views/widget/widget_delegate.h
@@ -170,6 +170,7 @@ class DesktopDataControlsDialog;
 }
 
 namespace electron {
+class AutofillPopupView;
 class DevToolsWindowDelegate;
 }
 
@@ -923,6 +924,7 @@ class VIEWS_EXPORT WidgetDelegateView : public WidgetDelegate, public View {
   View* GetContentsView() override;
 
  private:
+  friend class electron::AutofillPopupView;
   // DO NOT ADD TO THIS LIST!
   // These existing cases are "grandfathered in", but there shouldn't be more.
   // See comments atop class.
